tests/bluetooth/bsim_bt/bsim_test_iso/src/main.c

```diff
@@ -106,12 +106,13 @@ static isoal_status_t test_sink_sdu_alloc(const struct isoal_sink    *sink_ctx,
 }
 
 
-static isoal_status_t test_sink_sdu_emit(const struct isoal_sink         *sink_ctx,
-					 const struct isoal_sdu_produced *valid_sdu)
+static isoal_status_t test_sink_sdu_emit(const struct isoal_sink             *sink_ctx,
+					 const struct isoal_emitted_sdu_frag *sdu_frag,
+					 const struct isoal_emitted_sdu      *sdu)
 {
-	printk("Vendor sink SDU len %u, seq_num %u, ts %u\n",
-		sink_ctx->sdu_production.sdu_written, valid_sdu->seqn,
-		valid_sdu->timestamp);
+	printk("Vendor sink SDU fragment len %u / %u, seq_num %u, ts %u\n",
+		sdu_frag->sdu_frag_size, sdu->total_sdu_size, sdu_frag->sdu.seqn,
```

Comment:
`printk` says "len", but variable is named "size". While these are very similar, I suggest to use same terminology